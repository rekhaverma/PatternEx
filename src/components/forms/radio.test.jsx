import React from 'react';
import { RadioButton } from './index';

describe('<RadioButton />', () => {
  const props = {
    'id': '',
    'onChange': () => null,
    'label': '',
    'name': '',
    'checked': false,
    'value': '',
    'className': '',
  };

  it('renders correctly', () => {
    const wrapper = shallow(<RadioButton {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
