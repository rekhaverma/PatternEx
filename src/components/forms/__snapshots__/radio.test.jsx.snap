// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RadioButton /> renders correctly 1`] = `
<div
  className="radio "
>
  <input
    checked={false}
    id=""
    name=""
    onChange={[Function]}
    type="radio"
    value=""
  />
  <label
    className="radio-label"
    htmlFor=""
  />
</div>
`;
